float64 komponenX
float64 komponenY
float64 komponenZ
---
float64 magnitude
